src/styles/header/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.nav {
+  display: flex;
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    color: #060b35;
+    font-size: 12px;
+    line-height: 60px;
+    font-weight: 500;
+    text-transform: uppercase;
+
+    &:hover {
+      color: var(--primary-color);
+      text-decoration: none;
+    }
+  }
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+ul {
+  margin: 0;
```

Comment:
It's recommended to reset default styles for lists (margin, padding, list-style-type) in a global reset file. This way, you don't have to reset them each time you use a list.